native/dgemm.go

```diff
@@ -165,48 +130,48 @@ func dgemmParallel(tA, tB blas.Transpose, a, b, c general64, alpha float64) {
 			alpha := alpha
 			aTrans := aTrans
 			bTrans := bTrans
-			crows := c.rows
-			ccols := c.cols
+			m := m
```

Comment:
These are functional lines. Circa 1.3 shared variables were always heap variables. This change here is just a renaming, deleting the lines is an actual functional change that should have benchmarks.